cmd/languageserver.go

```diff
@@ -26,19 +25,27 @@ func init() {
 			defer cancel()
 
 			opts := &lsp.LanguageServerOptions{
-				ErrorLog:       os.Stderr,
-				VerboseLogging: verboseLogging,
+				ErrorLog: os.Stderr,
 			}
 
 			ls := lsp.NewLanguageServer(opts)
 
-			conn := jsonrpc2.NewConn(
-				ctx,
-				jsonrpc2.NewBufferedStream(lsp.StdOutReadWriteCloser{}, jsonrpc2.VSCodeObjectCodec{}),
-				jsonrpc2.HandlerWithError(ls.Handle),
-			)
+			loggingCfg := lsp.ConnectionLoggingConfig{
+				Writer:      os.Stderr,
+				LogInbound:  false,
+				LogOutbound: false,
+			}
+			if verboseLogging {
+				loggingCfg.LogInbound = true
+				loggingCfg.LogOutbound = true
+			}
```

Comment:
[golf]
```suggestion
			loggingCfg := lsp.ConnectionLoggingConfig{
				Writer:      os.Stderr,
				LogInbound:  verboseLogging,
				LogOutbound: verboseLogging,
			}
```

😉 Like this, we could even inline it below.

Reply:
Ahh, now that looks better! Done in 5541657aa7c8252a1ddd3ce3bfe8932250fdc095